FrontEndLib/BitmapManager.h

```diff
@@ -68,6 +68,15 @@ struct CustomTiles
 	UINT wTileStartNo;     //starting number of these custom tiles
 };
 
+struct TileMask
+{
+	TileMask(UINT tile, int xOffset, int yOffset)
+		: tile(tile), xOffset(xOffset), yOffset(yOffset)
+	{ }
+	UINT tile;
+	int xOffset, yOffset; //tile location offset from top-left origin
```

Comment:
Top-left origin meaning top-left of the room widget? surface? screen? tile?

I am guessing tile, can you add a tiny comment at the top of the struct explaining how it's supposed to be used?

Reply:
Good call out.  I've clarified the comment:
`//relative tile location offset from top-left origin of the area this mask is being applied to`